pkg/services/services_test.go

```diff
@@ -0,0 +1,54 @@
+package services
```

Comment:
what does this file test?

Reply:
It was there to explore how the interfaces could be implemented and used. Its not longer needed, so they are removed.